test/src/e2e/informer.test.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import { expect } from "chai";
+import "mocha";
+import { wait } from "../helper/promise";
+import CodeChain from "../helper/spawn";
+
+describe("Informer", function() {
+    let nodeA: CodeChain;
+    let nodeB: CodeChain;
+    const WebSocket = require("ws");
```

Comment:
How about adding `informerWs` in the CodeChain type?
It will be greate if we can use websocket `nodeA.informerWS` like `nodeA.rpc`.

Reply:
@MSNTCS 
The comment below is not applied:
> How about adding informerWs in the CodeChain type?
It will be great if we can use websocket nodeA.informerWS like nodeA.rpc.
Do you have any reason to ignore the comment?

Reply:
Actually, because I did it 2 months ago, I do not remember why. I will try to reflect what you are saying.

Reply:
@majecty Do you want me to create a new repository like the foundry-rpc-js repository?

Reply:
No, that's not my intention.
What I said before is moving WebSocket initialization to `CodeChain` type: https://github.com/CodeChain-io/foundry/blob/49035aee1af5f191ff4581837003667a070a70ee/test/src/helper/spawn.ts#L69

Reply:
Done